pkg/config/default_test.go

```diff
@@ -1,11 +1,39 @@
 package config
 
 import (
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func TestDefaultMaskedPaths(t *testing.T) {
+	maskedPaths, err := getMaskedPaths()
+	assert.NoError(t, err)
+
+	root := "/sys/devices/system/cpu"
+
+	entries, err := os.ReadDir(root)
+	assert.NoError(t, err)
+
+	for _, entry := range entries {
+		if !strings.HasPrefix(entry.Name(), "cpu") {
+			continue
+		}
+		path := filepath.Join(root, entry.Name(), "thermal_throttle")
+		if _, err := os.Stat(path); err != nil {
+			if os.IsNotExist(err) {
+				continue
+			}
+			assert.NoError(t, err)
+		}
+		assert.True(t, slices.Contains(maskedPaths, path))
```

Comment:
this results in a bad expected true to be false message.
use assert.Contains() instead as this will include the paths in the error output so it is easy to see any obvious error